libs/librepcb/project/schematics/items/si_netlabel.h

```diff
@@ -65,6 +65,7 @@ class SI_NetLabel final : public SI_Base, public SerializableObject {
   const Angle&   getRotation() const noexcept { return mRotation; }
   SI_NetSegment& getNetSegment() const noexcept { return mNetSegment; }
   NetSignal&     getNetSignalOfNetSegment() const noexcept;
+  Length getWidth() noexcept;
```

Comment:
As the width is not an actual property of the netlabel, and is even not very exact, I would suggest to rename the method to something like `getApproximateWidth()`.

```suggestion
  Length getApproximateWidth() noexcept;
```